pkgs/development/python-modules/jupyterhub/default.nix

```diff
@@ -0,0 +1,55 @@
+{ lib
+, python
+, buildPythonPackage
+, fetchPypi
+, alembic
+, ipython
+, jinja2
+, python-oauth2
+, pamela
+, sqlalchemy
+, tornado
+, traitlets
+, requests
+, nodejs-8_x
+}:
+
+buildPythonPackage rec {
+  pname = "jupyterhub";
+  version = "0.8.1";
+  name = "${pname}-${version}";
```

Comment:
Since https://github.com/NixOS/nixpkgs/commit/78ed9da5e4d9c4cd3451d10195f816eb0c0983d8 name defaults to pname + version so this is not required anymore.